src/Sign.Cli/Resources.resx

```diff
@@ -135,6 +135,9 @@
   <data name="CodeCommandDescription" xml:space="preserve">
     <value>Sign binaries and containers.</value>
   </data>
+  <data name="CredentialTypeOptionDescription" xml:space="preserve">
+    <value>Azure credential type that will be used. This defaults to all types except interactive-browser, shared-token and visual-studio-code.</value>
```

Comment:
I don't think we need to hyphenate, since these are explicitly not supported values.

```suggestion
    <value>Azure credential type that will be used. This defaults to all types except interactive browser, shared token, and Visual Studio Code.</value>
    <
```

Reply:
Thanks for improving this message!